pkgs/build-support/kernel/modules-closure.nix

```diff
@@ -3,13 +3,14 @@
 # the modules identified by `rootModules', plus their dependencies.
 # Also generate an appropriate modules.dep.
 
-{ stdenvNoCC, kernel, firmware, nukeReferences, rootModules
-, kmod, allowMissing ? false }:
+{ writers, stdenvNoCC, kernel, firmware, rootModules
```

Comment:
This is not using `writers.writeRust` anymore?

Reply:
indeed, that was left over